tests/Control/UserDefinedFormControllerTest.php

```diff
@@ -367,4 +372,56 @@ public function testRecipientSubjectMergeFields()
         // check emails
         $this->assertEmailSent('test@example.com', 'no-reply@example.com', 'Email Subject: Basic Value');
     }
+
+    /**
+     * Note: will temporarily create an image in the websites public/assets/.protected folder
+     */
+    public function testImageThumbnailCreated()
+    {
+        // ensure framework is installed and a functioning website exists so there's somewhere
+        // for the uploaded file to go
+        if (!class_exists(Upload_Validator::class) || !file_exists(ASSETS_PATH)) {
+            $this->markTestSkipped("This test requires a functioning website");
+            return;
+        }
+
+        Config::modify()->set(Upload_Validator::class, 'use_is_uploaded_file', false);
+
+        $userform = $this->setupFormFrontend('upload-form');
+        $controller = UserDefinedFormController::create($userform);
+        $field = $this->objFromFixture(EditableFileField::class, 'file-field-1');
+
+        $path = realpath(__DIR__ . '/fixtures/testfile.jpg');
+        $data = [
+            $field->Name => [
+                'name' => 'testfile.jpg',
+                'type' => 'image/jpeg',
+                'tmp_name' => $path,
+                'error' => 0,
+                'size' => filesize($path),
+            ]
+        ];
+        $_FILES[$field->Name] = $data[$field->Name];
+
+        $controller->getRequest()->setSession(new Session([]));
+        $controller->process($data, Form::create());
```

Comment:
That might fix your 4.3 build.

```suggestion
        $controller->process($data, $controller->Form());
```

Reply:
Yup that fixed 4.3.x-dev